lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -173,7 +173,7 @@ def configure(conf)
 
       raise Fluent::ConfigError, "'max_retry_putting_template' must be positive number." if @max_retry_putting_template < 0
 
-      if @template_name && @template_file
+      if @template_name && @template_file && !Fluent::Engine.dry_run_mode
```

Comment:
Should we also add `&& !Fluent::Engine.dry_run_mode` clause for the following elsif statement?

```diff
+      elsif @templates && !Fluent::Engine.dry_run_mode
-      elsif @templates
         retry_operate(@max_retry_putting_template) do
           templates_hash_install(@templates, @template_overwrite)
         end
       end
```

Reply:
Indeed, I've pushed a new version that just wraps that whole if/elif block in `if !Fluent::Engine.dry_run_mode`.